src/webserver/WsBase.h

```diff
@@ -358,8 +341,7 @@ class WsSession : public WsBase,
         std::string responseStr = boost::json::serialize(response);
         if (!dosGuard_.add(*ip, responseStr.size()))
         {
-            warnings.emplace_back("Too many requests");
-            response["warnings"] = warnings;
+            response["warning"] = "load";
```

Comment:
What is this doing?

Reply:
This is the API convention for rate-limiting response stated on the doc here: https://xrpl.org/response-formatting.html#response-formatting